api/identitatem/v1alpha1/strategy_types.go

```diff
@@ -14,16 +16,40 @@ type StrategySpec struct {
 	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
 	// Important: Run "make generate-clients" to regenerate code after modifying this file
 
-	// Foo is an example field of Strategy. Edit strategy_types.go to remove/update
-	Foo string `json:"foo,omitempty"`
+	// Strategy to use for applying the AuthRealm to the managed clusters
+	// +kubebuilder:validation:Enum=backplane;grc
+	// +required
+	StrategyType StrategyType `json:"type"`
+
+	// Reference to a Placement CR created by the Strategy operator
+	// reference to a Placement CR created by the strategy operator that is a copy of the placement (Placement)
+	//   referenced in the ownerReference (AuthRealm) object's placement property,
+	//   patched with additional predicates to filter managed clusters by supported strategy type.
+	PlacementRef corev1.LocalObjectReference `json:"placementRef,omitempty"`
 }
 
+type StrategyType string
+
+const (
+	BackplaneStrategyType StrategyType = "backplane"
+	GrcStrategyType       StrategyType = "grc"
+)
+
 // StrategyStatus defines the observed state of Strategy
 type StrategyStatus struct {
 	// INSERT ADDITIONAL STATUS FIELD - define observed state of cluster
 	// Important: Run "make generate-clients" to regenerate code after modifying this file
+
+	// Conditions contains the different condition statuses for this AuthRealm.
+	// +optional
+	Conditions []metav1.Condition `json:"conditions,omitempty"`
 }
 
+const (
+	StrategyFailed  string = "Failed"
+	StrategySucceed string = "Succeed"
```

Comment:
This is the type of the conditions and so no need for the the Failed, the status of the "succeed" type condition will be false or true with a reason.
Also usually all types values are in lower cases: "succeed" and not "Succeed".

Reply:
I was looking at what Hive had done since that is the component I am most familiar with.  But I can go with lowercase.  Will remove Failed.

Reply:
I think this will need to be reworked as we see what status/conditions we want to surface.

Reply:
yes... for the time being is like just a placeholder